yoti/js/browser-loader.js

```diff
@@ -0,0 +1,19 @@
+(function(){
+    // Create browser script tag.
+    var script = document.createElement('script');
+    script.type='text/javascript';
+    script.async='async';
+    script.src='https://sdk.yoti.com/clients/browser.2.1.0.js';
```

Comment:
2.2.0 is latest.
2.3.0 (or 2.2.1) is coming very soon

Reply:
I'll change this to 2.2.0 for now and create tickets to update all plugins to 2.3.0 (or 2.2.1) when that's released